ligo/types.mligo

```diff
@@ -27,8 +80,13 @@ type tick_map = (tick_index, tick_state) big_map
 (* Position types, representing LP positions. *)
 type position_index = {owner : address ; lo : tick_index ; hi : tick_index}
 type position_state = {
+    (* Position's liquidity. *)
     liquidity : nat ;
+    (* Total fees earned by the position at the moment of last fees collection for this position.
+        This helps to evaluate the next portion of fees to collect.
+    *)
     fee_growth_inside_last : balance_nat ;
+    (* TODO: fill description once this field is used. *)
```

Comment:
This field will need to be deleted (because it should be calculated on-the-fly by an onchain view).

I guess we could take this chance to delete it while we're here? ^^

Reply:
Fair, let's remove it.